### ISSUE TITLE
PPL on wmt - 17

### ISSUE THREAD

**Issue by luffycodes** [jadore801120/attention-is-all-you-need-pytorch#152]
Hi, I'm sorry if this has been asked ! I tried to search in the closed issues, but did not find a related question.
Can I please get access to the .log file to check the final PPL and accuracy scores? The graph in readme does not have such a fine resolution.
Also, do we have to further fine-tune the parameters to get PPL scores achieved in Vaswani "Attention is all you need" paper, or these results are same as that achieved by the authors in the paper? I'm confused because the graph does not show the exact final PPL score at convergence.

Thanks a ton for the code :) It has been very helpful :+1:
